tools/make/lib/docs/api.mk

```diff
@@ -16,6 +16,22 @@
 # limitations under the License.
 #/
 
+# VARIABLES #
+
+# Branch/tag to clone and build docs for:
+STDLIB_BRANCH ?= develop
+
+# Compute the output directory name from the branch:
+ifeq ($(STDLIB_BRANCH),develop)
+	STDLIB_DOCS_DIRNAME := latest
```

Comment:
The convention we use elsewhere for this is `_FOLDER`. E.g., https://github.com/stdlib-js/stdlib/blob/01ab29036205f4a0507133c1a4a7fd429bb7bd9e/Makefile#L96. May be good to be consistent here.